frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-section-card/feature-flag-root-section-card.component.html

```diff
@@ -0,0 +1,14 @@
+<app-common-section-card>
+  <app-common-section-card-search-header 
+    header-left
+    [filterOptions]="['All', 'Name', 'Status', 'Context']"
+    [initialFilterOption]="'All'"
+    [initialSearchValue]=""
+    (search)="onSearch($event)"
+  ></app-common-section-card-search-header>
+  <div header-right>right</div>
+  <div content *ngIf="!(isLoadingFeatureFlags$ | async); else loading"><div content *ngIf="!(isLoadingFeatureFlags$ | async); else loading"><pre>{{( allFeatureFlags$ | async) | json }}</pre></div></div>
+  <ng-template #loading>
```

Comment:
why `*ngIf="!(isLoadingFeatureFlags$ | async); else loading"` condition div is here twice?